src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from 'react-dom';
+import App from './components/app/App';
+import { ThemeProvider } from './context/ThemeContext';
+
+render(
+  <ThemeProvider>
```

Comment:
Oh wait you did. Nice! Yeah so your Theme component isn't necessary if you do this